testutil/testdata/buf.gen.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
```

Comment:
we need to put this in testutil/testdata/v2 - the existing gogo test data needs to stay and then we want to add new stuff for pulsar